common/views/components/LabelsList/LabelsList.js

```diff
@@ -38,7 +41,7 @@ const LabelsList = ({
       >
         <Label
           label={label}
-          labelColor={labelColor}
+          labelColor={label.labelColor || labelColor}
```

Comment:
So `labelColor` is a fallback?

Reply:
more like labelColor is default - most list have the same colour labels, but on occasion we need to override one

Reply:
Ah, OK. Should we call it `defaultLabelColor` or something similar?